src/browser/core_state.ts

```diff
@@ -765,6 +765,16 @@ export function getInfoByUuidFrame(targetIdentity: Shapes.Identity): Shapes.Fram
         } else {
             writeToLog(1, `unable to find openfinWindow of child of ${app.uuid}`, true);
         }
+    } for (const ofView of app.views) {
```

Comment:
Nitpick: Can you move this `for` off this line please? Was having trouble following the curly braces